packages/contracts/scripts/deploy-scripts/mainnet.sh

```diff
@@ -0,0 +1,53 @@
+### All available deploy options at the time of deployment: ###
+#  --ctc-force-inclusion-period-seconds  Number of seconds that the sequencer has to include transactions before the L1 queue. (default: 2592000)
+#  --ctc-max-transaction-gas-limit       Max gas limit for L1 queue transactions. (default: 11000000)
+#  --deploy-scripts                      override deploy script folder path
+#  --em-max-gas-per-queue-per-epoch      Maximum gas allowed in a given queue for each epoch. (default: 250000000)
+#  --em-max-transaction-gas-limit        Maximum allowed transaction gas limit. (default: 11000000)
+#  --em-min-transaction-gas-limit        Minimum allowed transaction gas limit. (default: 50000)
+#  --em-ovm-chain-id                     Chain ID for the L2 network. (default: 420)
+#  --em-seconds-per-epoch                Number of seconds in each epoch. (default: 0)
+#  --export                              export current network deployments
+#  --export-all                          export all deployments into one file
+#  --gasprice                            gas price to use for transactions
+#  --l1-block-time-seconds               Number of seconds on average between every L1 block. (default: 15)
+#  --no-compile                          disable pre compilation
+#  --no-impersonation                    do not impersonate unknown accounts
+#  --ovm-address-manager-owner           Address that will own the Lib_AddressManager. Must be provided or this deployment will fail.
+#  --ovm-proposer-address                Address of the account that will propose state roots. Must be provided or this deployment will fail.
+#  --ovm-relayer-address                 Address of the message relayer. Must be provided or this deployment will fail.
+#  --ovm-sequencer-address               Address of the sequencer. Must be provided or this deployment will fail.
+#  --reset                               whether to delete deployments files first
```

Comment:
can we specify whether these are default on or off?  Are they all off if omitted?

Reply:
Yeah I believe they are all off if omitted